pkg/ipc/ipc.go

```diff
@@ -378,13 +377,11 @@ func (env *Env) parseOutput(p *prog.Prog) (*ProgInfo, error) {
 
 func convertExtra(extraParts []CallInfo) CallInfo {
 	var extra CallInfo
-	extraCover := make(cover.Cover)
 	extraSignal := make(signal.Signal)
 	for _, part := range extraParts {
-		extraCover.Merge(part.Cover)
 		extraSignal.Merge(signal.FromRaw(part.Signal, 0))
+		extra.Cover = append(extra.Cover, part.Cover...)
```

Comment:
This now does not deduplicate extra cover across parts. I will sleep better if we don't change behavior in the non-debug mode. You could look at the DedupCover flag here and de-dup the slice at the end if the flag is set.